packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "@okta/courage": "4.6.0-beta.2098.gc581fa3",
+    "@okta/courage": "4.6.0-alpha.2340.g2dc3cd2",
```

Comment:
just an reminder that don't forget to change to beta before merge.

Reply:
yup, will do!